Sources/TokamakStaticHTML/Modifiers/LayoutModifiers.swift

```diff
@@ -101,3 +101,13 @@ extension _PaddingLayout: DOMViewModifier {
       .joined(separator: " ")]
   }
 }
+
+extension _ShadowLayout: DOMViewModifier {
+  public var attributes: [HTMLAttribute: String] {
+    [
+      "style": "box-shadow: \(x)px \(y)px \(radius * 2)px 0px \(color.cssValue(.defaultEnvironment));",
```

Comment:
I almost forgot about this `defaultEnvironment`. I'm not sure how to make this come from the app's actual environment.

Reply:
I think you can add `@Environment(\.self) var environmentValues` to the modifier to access it.

Reply:
I wasn't able to use `@Environment(\.self)`. I used `EnvironmentReader` which is how this is done in other modifiers (specifically `_BackgroundModifier`).